lab2.js

```diff
@@ -48,7 +48,24 @@ function assert(expression, failureMessage) {
  with Dowington.
 */
 
-var hoursSpentInDowington; // TODO: assign me the value of the
+function Blob() {
+  this.lifespan = 0;
+  this.rate = 0;
+}
+
+Blob.prototype.consumption = function() {
+  var time = 1;
+  var i = 0;
+  while (i <= 1000) {
```

Comment:
What's 1000? I would pass this in to consumption rather than hard-coding it in there. Try to make your functions and methods as reusable as possible.